src/pharmpy/plugins/nlmixr/create_ini.py

```diff
@@ -0,0 +1,128 @@
+import pharmpy.model
+from pharmpy.deps import sympy
+from pharmpy.modeling import get_sigmas
+
+from .CodeGenerator import CodeGenerator
+from .name_mangle import name_mangle
+
+
+def add_theta(model: pharmpy.model.Model, cg: CodeGenerator) -> None:
+    """
+    Add THETAs to code generator when converting a model
+
+    Parameters
+    ----------
+    model : pharmpy.model.Model
+        Pharmpy model object.
+    cg : CodeGenerator
+        Code generator to add code upon.
+
+    Returns
```

Comment:
Remove Returns since it doesn't return anything.

Reply:
Removed